email-auditing-api.yaml

```diff
@@ -0,0 +1,253 @@
+  title: Email Auditing API
+  version: 0.0.2
+servers:
+  - url: https://{api-gateway-hostname}
+    variables:
+      api-gateway-hostname:
+        default: instance-tenant.apis.medallia.com
+        description:
+          The Medallia Experience Cloud hostname used to access Medallia Audit Email API
+paths:
+  /audit-email/v0/emails/tarfile:
+    get:
+      description: |
+        Downloads Invitation, Reminder, and/or Feedback (Rapid) response emails in a compressed (.tar.gz) file within a set period of time.
+      operationId: downloadTar
+      parameters:
+        - description:
+            The oldest timestamp from which emails are downloaded. Timestamp is in second granularity and is inclusive (for example, 12:00:01 includes the first second of the minute). Timestamp must be in YYYY-MM-DDTHH:mm:ssZ format (ISO 8601).
+          in: query
+          name: since
+          required: true
+          schema:
+            format: date-time
+            type: string
+        - description:
+            The most recent timestamp from which emails are downloaded. Timestamp is exclusive (for example, 12:00:01 excludes the first second of the minute). Timestamp must be in YYYY-MM-DDTHH:mm:ssZ format (ISO 8601).
+          in: query
+          name: until
+          required: true
+          schema:
+            format: date-time
+            type: string
+        - description: List of the Experience programs or Ad Hoc surveys IDs (separated by comma) emails are part of.
+          in: query
+          name: program_ids
+          schema:
+            type: string
+      responses:
+        "200":
+          content:
+            application/octet-stream:
+              schema:
+                format: binary
+                type: string
+          description: A compressed file (.tar.gz) with the emails to audit within the defined period of time (Since - Until).
+        "401":
+          content:
+            application/octet-stream:
+              schema:
+                $ref: "#/components/schemas/error"
+          description: Unauthorized.
+        "403":
+          content:
+            application/octet-stream:
+              schema:
+                $ref: "#/components/schemas/error"
+          description: Forbidden.
+        "500":
+          content:
+            application/octet-stream:
+              schema:
+                $ref: "#/components/schemas/error"
+          description: Internal server error.
+  /audit-email/v0/programs:
+    get:
+      description: |
+        Lists all the Experience programs or Ad Hoc surveys to audit emails from.
+      operationId: listPrograms
+      parameters:
+        - description: Position of the Experience program or Ad Hoc survey in the list.
```

Comment:
`position` is not intuitive as to what you're meaning.  This is a "page offset"?

Reply:
Yes, but as the name of the param is Offset didn't want to end up in a tautology.